### PR TITLE
Fixes master lol

### PR DESCRIPTION
**Pull request by JohnFulpWillard**

### CODE REVIEW

File: code/modules/shuttle/computer.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	if (HAS_TRAIT(user, TRAIT_FORBID_MINING_SHUTTLE_CONSOLE_OUTSIDE_STATION) && !is_station_level(user.z)) //Free golems and other mobs with this trait will not be able to use the shuttle from outside the station Z
 		to_chat(user, span_warning("You get the feeling you shouldn't mess with this."))
 		return
-	if(!user.can_read(src, check_for_light = FALSE)) //Illiterate mobs which aren't otherwise blocked from using computers will send the shuttle to a random valid destination
+	if(!user.can_read(src, reading_check_flags = (READING_CHECK_LITERACY))) //Illiterate mobs which aren't otherwise blocked from using computers will send the shuttle to a random valid destination
```
Comment (Fikou) [inline code/modules/shuttle/computer.dm:47]: why the parentheses

**Comment by JohnFulpWillard** [comment]
merge skew between https://github.com/tgstation/tgstation/pull/69924 and https://github.com/tgstation/tgstation/pull/69836

File: code/modules/shuttle/computer.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/modules/shuttle/computer.dm:47]: i like them

Comment (Fikou) [reply code/modules/shuttle/computer.dm:47]: they r stupid for flags

Comment (JohnFulpWillard) [reply code/modules/shuttle/computer.dm]: u r stupid

**Review by Fikou — APPROVED** [review]
will merge when checks pass

**Pull request merged**